package.json

```diff
@@ -4,7 +4,7 @@
   "description": "react component for rendering formatted timestamps",
   "main": "lib/index.js",
   "peerDependencies": {
-    "react": "^0.14.0-rc1 || >= 0.13.0, < 0.15.0",
+    "react": ">= 0.13.0",
```

Comment:
I still think having a higher bound is good as we don't publish a component without making sure it works with the newer versions of React.